Add complete Simplified Chinese language file.

Tianary opened the pull request:
Add a complete Simplified Chinese language file. Hopefully it will be merged into subsequent releases, which will help some Chinese players to use the mod easily. Also, I see in PR#74 that the 1.13.2+ branch is behind the master branch, which means more features will be added in subsequent releases? If so, I will update the language file after that.

Tianary committed:
Add zh_cn.json

Add Simplified Chinese language file.

maruohon commented:
Yes, all the current 1.13.2+ versions of the client mods are based on main development code from early June 2019, so they are pretty much a year behind by now. I'm currently trying to work on some malilib changes and additions that I want to finish before the next merge, so that I can hopefully start the next merge for all the mods at some point in the near-ish future.

Tiannary commented:
Got it, I'll update this translation when the next merger occurs, thanks.

Tiannary commented:
> Yes, all the current 1.13.2+ versions of the client mods are based on main development code from early June 2019, so they are pretty much a year behind by now. I'm currently trying to work on some malilib changes and additions that I want to finish before the next merge, so that I can hopefully start the next merge for all the mods at some point in the near-ish future.

Got it, I'll update this translation when the next merger occurs, thanks.

Tianary committed:
Fix some translation errors.

Fixed some translation errors in "修复 -> 鞘翅修复", "配置/激活热键 -> 复制告示牌文本" and "配置/激活热键 -> 鞘翅飞行自由视角".
